### PR TITLE
[10.0][RM] conflicts key from manifest file - product_categ_image

### PR DESCRIPTION
**Pull request by bodedra**

### CODE REVIEW

**Comment by bodedra** [comment]
@rousseldenis

File: product_categ_image/__manifest__.py [hunk 1]
```diff
@@ -13,7 +13,6 @@
     'author': 'Akretion,Odoo Community Association (OCA)',
     'website': 'https://github.com/OCA/product-attribute',
     'depends': ['product'],
-    'conflicts': ['product_default_image'],
```
Comment (pedrobaeza) [inline product_categ_image/__manifest__.py:16]: It should be `excludes`, but it will be active only since 11.0

**Comment by bodedra** [comment]
@pedrobaeza Comment addressed. Thanks for your review. :+1:

**Review by pedrobaeza — APPROVED** [review]

**Review by rousseldenis — APPROVED** [review]

**Review by rafaelbn — APPROVED** [review]
:+1:

**Comment by bodedra** [comment]
@OCA/product-maintainers This is ready for merge.

**Comment by pedrobaeza** [comment]
Well, in 10.0 it doesn't have effect, but can be orientative. Let's see if the branch results green: /ocabot merge

**Pull request merged**